linux-cachyos-bmq/PKGBUILD

```diff
@@ -311,6 +311,9 @@ prepare() {
     echo "Setting config..."
     cp ../config .config
 
+    echo "Setting config..."
```

Comment:
Same

Reply:
Clearly, added by mistake. There will be a correction in a moment

Reply:
Done